Cargo.toml

```diff
@@ -38,14 +38,17 @@ tokio-io = "0.1"
 [target.'cfg(not(any(target_os = "emscripten", target_os = "unknown")))'.dependencies]
 libp2p-dns = { version = "0.2.0", path = "./transports/dns" }
 libp2p-mdns = { version = "0.2.0", path = "./misc/mdns" }
+libp2p-noise = { version = "0.1.0", path = "./protocols/noise" }
 libp2p-tcp = { version = "0.2.0", path = "./transports/tcp" }
 
 [target.'cfg(any(target_os = "emscripten", target_os = "unknown"))'.dependencies]
 stdweb = { version = "0.4", default-features = false }
 
 [dev-dependencies]
 env_logger = "0.6.0"
+quicli = "0.4"
```

Comment:
I suppose that this and `structopt` are no longer needed anymore, since the example is gone.